### ISSUE TITLE
What's the effect of the hyperparameter max_real in multiview reconstruction?

### ISSUE THREAD

**Issue by zhejia01** [SonSang/dmesh#10]
Now that value of points_real is normalized to [0,1] in each step, why not just set the max_real to 1.0 instead of 0.003?

**Comment by SonSang** [comment]
Hi, sorry for the late reply. That was a little trick to make the optimization speed of `points_real` to be faster than the other variables (that is, making learning rate of `points_real` larger). We found out that making `points_real` to change faster than the other variables, like point positions or weights, helps faster convergence.

**Comment by zhejia01** [comment]
I get it! Thanks!
